### ISSUE TITLE
System.IO.DirectoryNotFoundException: Could not find a part of the path '/app/Logs'.

### ISSUE THREAD

**Issue by mregni** [mregni/EmbyStat#600]
View details in Rollbar: [https://rollbar.com/UPing/EmbyStat/items/181/](https://rollbar.com/UPing/EmbyStat/items/181/)

```
Traceback (most recent call last):
  File "System.IO.Enumeration.FileSystemEnumerator`1", line 42, in System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(System.String path, System.Boolean ignoreNotFound)
  File "System.IO.Enumeration.FileSystemEnumerator`1", line 56, in System.IO.Enumeration.FileSystemEnumerator`1..ctor(System.String directory, System.IO.EnumerationOptions options)
  File "System.IO.Enumeration.FileSystemEnumerable`1", line 66, in System.IO.Enumeration.FileSystemEnumerable`1..ctor(System.String directory, System.IO.Enumeration.FileSystemEnumerable`1+FindTransform[TResult] transform, System.IO.EnumerationOptions options)
  File "System.IO.Enumeration.FileSystemEnumerableFactory", line 63, in System.IO.Enumeration.FileSystemEnumerableFactory.UserFiles(System.String directory, System.String expression, System.IO.EnumerationOptions options)
  File "System.IO.Directory", line 97, in System.IO.Directory.InternalEnumeratePaths(System.String path, System.String searchPattern, System.IO.SearchTarget searchTarget, System.IO.EnumerationOptions options)
  File "System.IO.Directory", line unknown, in System.IO.Directory.EnumerateFiles(System.String path)
  File "/EmbyStat.Services/LogService.cs", line 26, in EmbyStat.Services.LogService.GetLogFileList()
  File "/EmbyStat.Controllers/Log/LogController.cs", line 25, in EmbyStat.Controllers.Log.LogController.GetLogFileLIst()
  File "(unknown)", line 45, in lambda_method(System.Runtime.CompilerServices.Closure , System.Object , System.Object[] )
  File "Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor+SyncActionResultExecutor", line unknown, in Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor+SyncActionResultExecutor.Execute(Microsoft.AspNetCore.Mvc.Infrastructure.IActionResultTypeMapper mapper, Microsoft.Extensions.Internal.ObjectMethodExecutor executor, System.Object controller, System.Object[] arguments)
  File "Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+<InvokeActionMethodAsync>d__12", line 198, in Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+<InvokeActionMethodAsync>d__12.MoveNext()
  File "System.Runtime.ExceptionServices.ExceptionDispatchInfo", line 12, in System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
  File "Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+<InvokeNextActionFilterAsync>d__10", line 165, in Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+<InvokeNextActionFilterAsync>d__10.MoveNext()
  File "System.Runtime.ExceptionServices.ExceptionDispatchInfo", line 12, in System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
  File "Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker", line 21, in Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(Microsoft.AspNetCore.Mvc.Filters.ActionExecutedContext context)
  File "Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker", line 905, in Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+State& next, Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+Scope& scope, System.Object& state, System.Boolean& isCompleted)
  File "Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+<InvokeInnerFilterAsync>d__13", line 47, in Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker+<InvokeInnerFilterAsync>d__13.MoveNext()
  File "System.Runtime.ExceptionServices.ExceptionDispatchInfo", line 12, in System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
  File "Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker+<InvokeNextExceptionFilterAsync>d__24", line 166, in Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker+<InvokeNextExceptionFilterAsync>d__24.MoveNext()
System.IO.DirectoryNotFoundException: Could not find a part of the path '/app/Logs'.
```

**Comment by mregni** [comment]
Looks like path to the Logs folder is wrong in some environments

**Comment by mregni** [comment]
Clossing the ticket because should be fixed with #601
